bioblend/galaxy/objects/wrappers.py

```diff
@@ -1250,6 +1251,21 @@ def run(self, inputs, history, wait=False,
         return outputs
 
 
+class Job(Wrapper):
+    """
+    Maps to a Galaxy job.
+    """
+    BASE_ATTRS = Wrapper.BASE_ATTRS
+    POLLING_INTERVAL = 10  # for output state monitoring
```

Comment:
I meant POLLING_INTERVAL. If you intend to add a `wait()` method in the future, then just change `s/output/job/` in the comment.

About BASE_ATTRS, I think it may be useful to add at least `'state'`.